tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake

```diff
@@ -30,4 +30,8 @@ PDM1_MIC_B_ENABLE=1,DMIC0_PCM_ID=99,PREPROCESS_PLUGINS=nhlt,\
 NHLT_BIN=nhlt-sof-mtl-max98357a-rt5682.bin,DEEPBUFFER_FW_DMA_MS=10,HEADSET_SSP_DAI_INDEX=2,\
 SPEAKER_SSP_DAI_INDEX=0,HEADSET_CODEC_NAME=SSP2-Codec,SPEAKER_CODEC_NAME=SSP0-Codec,\
 BT_NAME=SSP1-BT,BT_INDEX=1,BT_ID=8,BT_PCM_NAME=Bluetooth,INCLUDE_ECHO_REF=true"
+
+"cavs-sdw\;sof-mtl-rt711-l0-rt1316-l23-rt714-l1\;PLATFORM=mtl,NUM_SDW_AMPS=2,SDW_DMIC=1,\
```

Comment:
I am also not very happy about NUM_SDW_AMPS=2 because it does not provide any indication that different links are used. This could also be set in the case of two amplifiers on the same link.

This seems very confusing @aiChaoSONG @bardliao

Reply:
> I am also not very happy about NUM_SDW_AMPS=2 because it does not provide any indication that different links are used. This could also be set in the case of two amplifiers on the same link.
> 
> This seems very confusing @aiChaoSONG @bardliao

Maybe rename it to **NUM_SDW_AMP_LINKS**?

Reply:
Ah, it's about the number of links or the number of amps? That's a nice confusion indeed :-)

Reply:
> Ah, it's about the number of links or the number of amps? That's a nice confusion indeed :-)

It is about number of links. If the 2 amps are on the same sdw link, NUM_SDW_AMPS should be 1 because we have only one cpu dai.

Reply:
ok, let's rename this then, thanks for the explanation @bardliao